packages/obojobo-pages-page/viewer-component.js

```diff
@@ -9,7 +9,7 @@ const { OboComponent } = Common.components
 const { NavUtil } = Viewer.util
 
 export default class Page extends React.Component {
-	componentWillReceiveProps(nextProps) {
+	UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
Could this be replaced with componentDidUpdate? That's the recommended React Method

Reply:
This may require https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops